src/vswhere/Program.cpp

```diff
@@ -12,6 +14,10 @@ void WriteLogo(_In_ const CommandArgs& args, _In_ wostream& out);
 
 int wmain(_In_ int argc, _In_ LPCWSTR argv[])
 {
+    _setmode(_fileno(stdout), _O_U16TEXT);
```

Comment:
I tried use `wcout.imbue` (http://stackoverflow.com/a/2557315/61505), but it doesn't work